src/hillslope_method.R

```diff
@@ -16,7 +16,7 @@ library('rgdal')
 
 #--------------------------------------------------------------------------------------
 # Own packages
-source("plot.R")
+source("plot_old.R")

```

Comment:
Here you import one of the `_old` functions, I think you should remove the usage of all `_old` stuff so that we can savely remove it in the future.